proteus/gmsh2triangle.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+from proteus.MeshTools import Edge
+from proteus.Profiling import logEvent
+
+def msh2triangle(fileprefix):
```

Comment:
Yes, it creates .ele .node and .edge files from a .msh file. I will add a docstring and move it to MeshTools.